tap_linkedin_ads/sync.py

```diff
@@ -367,7 +367,7 @@ def sync(client, config, catalog, state):
             'bookmark_field': 'last_modified_time',
             'id_fields': ['id', 'reference_organization_id'],
             'children': {
-                'video_ads': {
+                'video_advertisment': {
```

Comment:
in general, this is probably not how we should fix it. Every company using this tap would suddenly have their data re-routed to a different table.